server.py

```diff
@@ -51,6 +70,11 @@ class Meta:
 
 
 class ContestProblems(Model):
+    """
```

Comment:
nothing new. Perhaps this docstring can explain WHY this model is necessary and what all features exist in PyJudge because of this model